hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -43,10 +48,27 @@ private OzoneManagerRatisUtils() {
    * @return OMClientRequest
    * @throws IOException
    */
-  public static OMClientRequest createClientRequest(OMRequest omRequest)
-      throws IOException {
+  public static OMClientRequest createClientRequest(OMRequest omRequest) {
     Type cmdType = omRequest.getCmdType();
     switch (cmdType) {
+    case CreateVolume:
+      return new OMVolumeCreateRequest(omRequest);
+    case SetVolumeProperty:
+      boolean hasQuota = omRequest.getSetVolumePropertyRequest()
+          .hasQuotaInBytes();
+      boolean hasOwner = omRequest.getSetVolumePropertyRequest().hasOwnerName();
+      Preconditions.checkState(hasOwner || hasQuota , "Either Quota or owner " +
+          "shuould be set in the SetVolumeProperty request");
+      Preconditions.checkState(!(hasOwner && hasQuota) , "Either Quota or " +
+          "owner shuould be set in the SetVolumeProperty request. Should not " +
+          "be set both");
```

Comment:
NIT: 1. typo in should
2.typo: not set both

Reply:
Done